.github/workflows/ci.yaml

```diff
@@ -107,10 +107,10 @@ jobs:
             bufbuild/buf
           tags: |
             type=edge,branch=main
-            type=semver,pattern={{major}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}.{{minor}}.{{patch}}
             type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
```

Comment:
This looks to match the examples here: https://github.com/docker/metadata-action/tree/902fa8ec7d6ecbf8d84d538b9b233a880e428804?tab=readme-ov-file#semver

Reply:
Do these two get de-dupe? `{{version}}` and `{{major}}.{{minor}}.{{patch}}`?

Reply:
Yeah - lemme remove. We shouldn't list them both times I don't think.

Reply:
Fixed in 145ec7d.